localstack/aws/handlers/cors.py

```diff
@@ -20,6 +20,7 @@
 from localstack.http import Response
 
 from ...constants import LOCALHOST, LOCALHOST_HOSTNAME, PATH_USER_REQUEST
+from ...utils.bootstrap import log_duration
```

Comment:
nit: ideally we don't use relative imports of modules that are outside the scope of the component we are building. basically: relative imports are used when things belong together in a component, absolute imports are used when importing code that represents a dependency to another component

Reply:
Right, I'm not sure why there are dynamic import here, I can change them 👍